### ISSUE TITLE
SwaggerUIBundle is not defined

### ISSUE THREAD

**Issue by viveksahu56722** [python-restx/flask-restx#549]
I am using `flask-restx==1.1.0`

My Python is `3.8.10`

Sometime I am seeing this issue in my swagger dashboard

`GET https://{host}/api/swaggerui/swagger-ui-standalone-preset.js net::ERR_ABORTED 404 (NOT FOUND)
{host}/:71     GET https://{host}/api/swaggerui/swagger-ui-bundle.js net::ERR_ABORTED 404 (NOT FOUND)
{host}/:7     GET https://{host}/api/swaggerui/swagger-ui.css net::ERR_ABORTED 404 (NOT FOUND)
(index):75 Uncaught ReferenceError: SwaggerUIBundle is not defined
   at window.onload ((index):75:40)`

And my dashboard is not getting load because of this issue

Can someone please help me with this ? I am not able to find much about this issue in internet already

**Comment by peter-doggart** [comment]
@viveksahu56722 Did you install `flask-restx` using pip? It sounds like the swagger components are not installed. 

If you clone the project directly from github, you need to have Invoke and npm installed and run `inv assets` to download the swagger components. We do this as part of the build process when it's published to pypi so they are included in the pip available version.

**Comment by viveksahu56722** [comment]
Yeah I have installed it using pip but this issue is not always there.
Sometimes it is there and automatically gets fixed but again may occur after some time, what could be causing it ? 
The above solution for this case too as sometime its working fine ?

**Comment by peter-doggart** [comment]
It randomly appears whilst the application is running? How is the application deployed?

**Comment by viveksahu56722** [comment]
Yes it has been appearing randomly, can it be because of API requests loads in server ? if there are very frequent calls ? 
Its deployed on VMware or Physical (Ubuntu 20.04) I think

**Comment by chipndell** [comment]
I agree @peter-doggart, seems like swagger files are not installed. I would recommand to start fresh with _new venv_ and _install all packages_. Let us know how that goes. :-)
